manifests/init.pp

```diff
@@ -96,6 +96,7 @@
   $keepalive_requests                                        = '100',
   $log_format                                                = {},
   Boolean $mail                                              = false,
+  Variant[String, Boolean] $mime_types_path                  = 'mime.types',
```

Comment:
```suggestion
  Variant[String[1], Boolean] $mime_types_path               = 'mime.types',
```

Reply:
I have added a check to the nginx.conf.erb template